internal/cmd/server.go

```diff
@@ -99,6 +100,10 @@ func defaultServerOptions(infraDir string) server.Options {
 			HTTPS:   ":443",
 			Metrics: ":9090",
 		},
+
+		Cache: cache.Options{
+			Port: 6379,
+		},
```

Comment:
We probably want to call this configuration `redis` as well, since redis may be used for different things, and at least initially is not used at all for caching.

Reply:
This has been updated locally after @ssoroka's comment